hack/testing/logging.sh

```diff
@@ -142,7 +142,7 @@ os::cmd::expect_success 'oadm router'
 ######### logging specific code starts here ####################
 
 os::cmd::expect_success "oadm new-project logging --node-selector=''"
-os::cmd::expect_success "oc project logging"
+os::cmd::expect_success "oc project logging > /dev/null"
```

Comment:
@ewolinetz correct - it just reduces the noise in our tests - has no impact on expect_success.